deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,90 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+/* #C16E70 old rose #F2F3D9 beige #030027 blue*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    width: 100vw;
+    height: 100vh;
+    flex-direction: column;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    background: #F2F3D9;
+    text-align: center;
+    flex-shrink: 1;
+    flex-grow: 0;
+}
+
+header {
+    display: flex;
+    flex-basis: 5vw;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #030027;
+    color: white;
+    flex-shrink: 0;
+    flex-grow: 0;
+
+}
+
+.header-h3{
+    margin: 40px;
+}
+
+ul{
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    gap: 50px;
+    margin: 40px;
+}
+
+div{
+    font-size: 15px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-basis: 200px;
+    justify-content: center;
+    gap: 25px;
+    flex-wrap: wrap;
+    flex-shrink: 0;
+    flex-grow: 0;
+
```

Comment:
het is goed om wat margin aan de zijkanten van je div toe te voegen, zodat als het scherm kleiner wordt de articles niet tegen de rand van het scherm aankomen.